src/game/index.js

```diff
@@ -7,6 +7,9 @@ const homeBoard = {
   tokens: [], // Floating tokens
   inventory: [
     'timber', // TODO remove
+    'timber',
+    'timber',
+    'timber',
```

Comment:
haha, great minds think alike. this definitely makes smoke testing easier. i'm down to leave it for now.